### PR TITLE
Clarify where and what Log Messages is 

### PR DESCRIPTION
**Pull request by DelazJ**
Because users often wonder what this "log messages" means and whether it's another panel in the processing alg dialog and how to activate it.
Also push error messages to translation as it's always good to understand what goes wrong.

### CODE REVIEW

File: python/plugins/processing/gui/Postprocessing.py [hunk 1]
```diff
@@ -78,15 +78,15 @@ def handleAlgorithmResults(alg, context, feedback=None, showResults=True):
...
         except Exception:
-            QgsMessageLog.logMessage("Error loading result layer:\n" + traceback.format_exc(), 'Processing', QgsMessageLog.CRITICAL)
+            QgsMessageLog.logMessage(QCoreApplication.translate('PostProcessing', "Error loading result layer:\n") + traceback.format_exc(), 'Processing', QgsMessageLog.CRITICAL)
             wrongLayers.append(str(l))
         i += 1
 
     QApplication.restoreOverrideCursor()
     if wrongLayers:
-        msg = "The following layers were not correctly generated.<ul>"
+        msg = QCoreApplication.translate('PostProcessing', "The following layers were not correctly generated.<ul>")
```
Comment (m-kuhn) [inline python/plugins/processing/gui/Postprocessing.py]: I would try to keep HTML codes or control sequences like `\n` out of translatable strings wherever possible

**Review by m-kuhn — APPROVED** [review]

**Comment by DelazJ** [comment]
@m-kuhn I'd be interested in tackling https://github.com/qgis/QGIS/blob/master/python/plugins/processing/tools/dataobjects.py#L164 but no idea what i could set for context. Any hint?

**Comment by DelazJ** [comment]
And any idea what the Processing framework log is? Is it the same Log Messages Panel? @alexbruy

**Comment by alexbruy** [comment]
@DelazJ Processing writes messages into QGIS message log. Also some information printed in the Log tab of the algorithm dialog

**Comment by nyalldawson** [comment]
Off topic language lesson :

The correct word to use in the title here is "clarify". I've seen this translation mistake made frequently in the qgis community in various strings and docs, so I'm guessing it's a tricky one coming from certain languages.

My hint would be: "never use precise, unless you're talking about an outdated Ubuntu version"!

/Lesson over

**Comment by m-kuhn** [comment]
> My hint would be: "never use precise, unless you're talking about an outdated Ubuntu version"!

Thanks for this precise clarification.

**Comment by DelazJ** [comment]
Fixed, Nyall

**Pull request merged**